iree/compiler/Conversion/LinalgToLLVM/LinalgBufferizePass.cpp

```diff
@@ -172,6 +173,53 @@ LogicalResult convertAnyLinalgOp(OpBuilder &b, linalg::LinalgOp op,
   return success();
 }
 
+static LogicalResult convertTransferOp(OpBuilder &b,
+                                       VectorTransferOpInterface op,
+                                       BlockAndValueMapping &bvm) {
+  if (op.getShapedType().isa<MemRefType>()) return success();
```

Comment:
We generally return failure when the pattern fails to replace the op.
Not sure whether returning success is correct or may trigger weird stuff down the line?